templates/publisher/publicise.html

```diff
@@ -39,8 +39,8 @@
             <div class="col-5">
               <p class="snapcraft-publicise__images">
                 {% set url = "images/badges/" + lang + "/snap-store-black.svg" %}
-                <a href="https://snapcraft.io/{{ snap_name }}" title="{{ data.text }}">
-                  <img src="{{ static_url(url) }}" />
+                <a href="https://snapcraft.io/{{ snap_name }}">
+                  <img alt="{{ data.text }} src="{{ static_url(url) }}" />
```

Comment:
```suggestion
                  <img alt="{{ data.text }}" src="{{ static_url(url) }}" />
```

Might want to close the attribute :smiley: